drivers/SmartThings/philips-hue/src/cosock/socket/internals.lua

```diff
@@ -0,0 +1,174 @@
+-- Copyright 2021 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+local m = {}
+local log = require "log"
+local unpack = table.unpack or unpack
+local pack = table.pack or pack or function(a, b, c, d, e)
+  -- This is a shim for lua 5.1 which doesn't provide a `pack` operation
```

Comment:
this _shouldn't_ be required for us since I think everything is on 5.3, right?

Reply:
That's my understanding as well, most likely just a holdover from the fact that cosock is from outside of ST and is targeting more Lua version compatibility.